client/src/styles/sidebar.css

```diff
@@ -0,0 +1,59 @@
+.sidebar-extended {
+  width: 200px;
+  height: 750px;
+  background-color: #012323;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.sidebar {
+  width: 50px;
+  height: 750px;
+  background-color: #012323;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: left;
+}
+
+.icon-container-extended {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  color: white;
+  padding: 3px;
+}
+
+.icon-container {
```

Comment:
instead of making a CSS class selector for display:flex, align-items, it might be easier to use the bootstrap class selectors in the future

https://getbootstrap.com/docs/4.0/utilities/flex/

for example:
<div id="icon-container" class="d-flex align-items-center justify-content-start text-white p-1"></div>